docs/src/callbacks.md

```diff
@@ -112,7 +112,7 @@ will yield the following plot:
 * The [`VisualizationCallback`](@ref) can be used for in-situ visualization. See
   [Visualizing results during a simulation](@ref).
 * The [`TrivialCallback`](@ref) does nothing and can be used to easily disable some callbacks
-  via [`trixi_include`](@ref).
+  via [`trixi_include`](https://trixi-framework.github.io/TrixiBase.jl/stable/reference/#TrixiBase.trixi_include-Tuple{Module,%20AbstractString}).
```

Comment:
We should discuss whether we want to keep the docs for TrixiBase.jl only there or whether we want to API reference also in the main Trixi.jl docs to ensure that we can still link directly via `@ref`

Reply:
Can we add external API to Trixi? Or are you talking about duplicating the docstring?

Reply:
I was thinking about adding an API section for TrixiBase.jl similar to what we do for Trixi2Vtk.jl right now at https://trixi-framework.github.io/Trixi.jl/stable/reference-trixi2vtk/
That should work, shouldn't it?

Reply:
That's cool. Didn't know that was possible. I'll do that.

Reply:
Great, thanks!